gulpfile.js

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable no-console */
+/* eslint-env node */
+
+'use strict';
+
+const { parallel, watch } = require('gulp');
+const { exec } = require('child_process');
+
+function build(cb) {
+  exec('make build', (error, stdout, stderr) => {
+    console.error(stderr);
+    console.log(stdout);
+    cb(error);
+  });
+}
+
+function watchClient() {
+  watch('node_modules/hypothesis', { events: 'all' }, build);
```

Comment:
Do we need `events: 'all'` here? According to https://gulpjs.com/docs/en/getting-started/watching-files/#watched-events the default behavior is to:

>  executes tasks whenever a file is created, changed, or deleted. 

That's probably adequate.

Reply:
You are probably right. Because `./node_module/hypothesis` is sometimes a soft link (because of `yarn link`) I wanted to catch the `unlink` event which is available in `events: all`:

> The available events are 'add', 'addDir', 'change', 'unlink', 'unlinkDir', 'ready', 'error'. Additionally 'all' is available, which represents all events other than 'ready' and 'error'.

Reply:
I confirm that `events: all` it is necessary for example to detect the addition of new files to it triggers the transpilation.